.github/stale.yml

```diff
@@ -2,10 +2,6 @@
 daysUntilStale: 60
 # Number of days of inactivity before a stale issue is closed
 daysUntilClose: 7
-# Issues with these labels will never be considered stale
-# exemptLabels:
- # - pinned
```

Comment:
Would it make sense to apply the same `  - dependencies` exemption here as in `mender-cli`?

Reply:
It is already there, line 15 :+1: I am just removing the "duplicate".

Reply:
argh sorry for the noise, I just looked at the github diff 🙈